test/e2e/logging_test.go

```diff
@@ -5,84 +5,78 @@ import (
 
 	operatorsv1 "github.com/openshift/api/operator/v1"
 	deploymentsub "github.com/openshift/console-operator/pkg/console/subresource/deployment"
-	"github.com/openshift/console-operator/pkg/testframework"
+	"github.com/openshift/console-operator/test/e2e/framework"
 )
 
-func setupLoggingTestCase(t *testing.T) *testframework.Clientset {
-	client := testframework.MustNewClientset(t, nil)
-	testframework.MustManageConsole(t, client)
-	testframework.MustNormalLogLevel(t, client)
-	return client
+func setupLoggingTestCase(t *testing.T) (*framework.ClientSet, *operatorsv1.Console) {
+	return framework.StandardSetup(t)
 }
 
-func cleanUpLoggingTestCase(t *testing.T, client *testframework.Clientset) {
-	testframework.WaitForSettledState(t, client)
+func cleanUpLoggingTestCase(t *testing.T, client *framework.ClientSet) {
+	framework.StandardCleanup(t, client)
 }
 
 // TestDebugLogLevel sets 'Debug' LogLevel on the console operator and tests
 // if '--log-level=*=DEBUG' flag is set on the console deployment
 func TestDebugLogLevel(t *testing.T) {
-	client := setupLoggingTestCase(t)
-	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	client, _ := setupLoggingTestCase(t)
```

Comment:
can't we just call directly ?
```suggestion
	client, _ := framework.StandardSetup(t)
```

Reply:
So there are recommendations that the setup func should return a func that is the cleanup func, like this:

```golang

func setup() {
   // do stuff...then return cleanup func
   return func() {
      // do cleanup, can still reference vars from closure. 
   }
}

func TestFeatureFoo(t *testing.T) {
   // setup() is called, then the defer is applied to the returned func()
   defer setup()()
}
```

I'd also like to see these in the form of table tests like our unit tests:

```golang
func TestFeatureFoo(t *testing.T) {
  	client, _ := setupFooTest(t)
	defer cleanupFooTest(t, client)
        tests := []struct {
		name string  // names would be nice, we are missing this right now
		input args
		output result	
         } {

         }
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {
                    // do some work, prob like noted above, call setup() and defer the func() for cleanup
                    defer setup()()
                    if FeatureFoo(tt.input) != tt.output {
                         t.Error("it broke")
                    }
		})
	}
}
```